package.json

```diff
@@ -167,6 +167,7 @@
     "lerna": "^8.1.2",
     "lodash": "^4.17.21",
     "markdownlint-cli2": "^0.13.0",
+    "markdownlint-rule-search-replace": "^1.2.0",
```

Comment:
If you move this to `"dependencies"` I believe Toolpad and X won't have to add this as adependency as well

Reply:
This feels wrong, but it is convenient 😅
I'll move it to dependencies, thanks!